extensions/git/src/timelineProvider.ts

```diff
@@ -200,7 +200,7 @@ export class GitTimelineProvider implements TimelineProvider {
 			if (working) {
 				const date = new Date();
 
-				const item = new GitTimelineItem('', index ? '~' : 'HEAD', localize('git.timeline.uncommitedChanges', 'Uncommitted Changes'), date.getTime(), 'working', 'git:file:working');
+				const item = new GitTimelineItem('', index ? '~' : 'HEAD', localize('git.timeline.uncommittedChanges', 'Uncommitted Changes'), date.getTime(), 'working', 'git:file:working');
```

Comment:
Changing the keys in localize calls will force the entries to be relocalized and users will loose thir translations until the new localizations come in. So please don't change these.